modules/engine/src/main/scala/observe/engine/Handle.scala

```diff
@@ -102,18 +102,18 @@ object Handle {
   }
 
   extension [F[_]: Functor, D, A](self: StateT[F, D, A])
-    def toHandle[V]: Handle[F, D, V, A] = Handle[F, D, V, A](self.map((_, None)))
+    def toHandleT[V]: Handle[F, D, V, A] = Handle[F, D, V, A](self.map((_, None)))
```

Comment:
Why the rename?

Reply:
The name collided in one use. It worked before because I used and alias in an import, but for some reason now the import triggers an "unused" warning.